### PR TITLE
Set camera_num for convenience

### PR DESCRIPTION
**Pull request by Terminus-IMRC**
It's useful on compute modules.

### CODE REVIEW

File: host_applications/linux/apps/raspicam/raspiraw.c [hunk 1]
```diff
@@ -46,6 +46,9 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include <sys/ioctl.h>
 
+//Camera to use
+#define CAMERA_NUM 1
```
Comment (6by9) [inline host_applications/linux/apps/raspicam/raspiraw.c]: Seeing as I had to check the default value, a comment along the lines of "Set to 1 on all Pi models except CM. On CM, 0=CAM0 connector, 1=CAM1 connector" would be useful.

File: host_applications/linux/apps/raspicam/raspiraw.c [hunk 2]
```diff
@@ -476,6 +479,13 @@ int main(int argc, char** args) {
 		goto component_destroy;
 	}
 
+	status = mmal_port_parameter_set_int32(output, MMAL_PARAMETER_CAMERA_NUM, CAMERA_NUM);
+	if(status != MMAL_SUCCESS)
+	{
+		vcos_log_error("Failed to set cfg");
```
Comment (6by9) [inline host_applications/linux/apps/raspicam/raspiraw.c]: Copy/paste on the error message from above.

**Comment by Terminus-IMRC** [comment]
Thank you for reviewing! I will do it.

**Comment by Terminus-IMRC** [comment]
Added another commit.

**Comment by 6by9** [comment]
Thanks.

**Pull request merged**